doc/rados/configuration/common.rst

```diff
@@ -100,75 +104,78 @@ Additionally, you should enable message signing. See `Cephx Config Reference`_ f
-end is now default, but when using Filestore you specify a journal size. For example:
+When Ceph production clusters deploy :term:`Ceph OSD Daemons`, the typical
+arrangement is that one node has one OSD daemon running Filestore on one
+storage device. BlueStore is now the default back end, but when using Filestore
+you must specify a journal size. For example:
 
 .. code-block:: ini
 
-	[osd]
-	osd_journal_size = 10000
+    [osd]
+    osd_journal_size = 10000
 
-	[osd.0]
-	host = {hostname} #manual deployments only.
+    [osd.0]
+    host = {hostname} #manual deployments only.
 
 
-By default, Ceph expects to store a Ceph OSD Daemon's data at the
-following path::
+By default, Ceph expects to store a Ceph OSD Daemon's data on the following
+path::
 
-	/var/lib/ceph/osd/$cluster-$id
+    /var/lib/ceph/osd/$cluster-$id
 
-You or a deployment tool (e.g., ``cephadm``) must create the corresponding
-directory. With metavariables fully expressed and a cluster named "ceph", this
-example would evaluate to::
+You or a deployment tool (for example, ``cephadm``) must create the
+corresponding directory. With metavariables fully expressed and a cluster named
+"ceph", the path specified in the above example evaluates to::
 
-	/var/lib/ceph/osd/ceph-0
+    /var/lib/ceph/osd/ceph-0
 
-You may override this path using the ``osd_data`` setting. We recommend not
-changing the default location. Create the default directory on your OSD host.
+You can override this path using the ``osd_data`` setting. We recommend that
+you do not change the default location. To create the default directory on your
+OSD host, run the following commands:
 
 .. prompt:: bash $
 
-	ssh {osd-host}
-	sudo mkdir /var/lib/ceph/osd/ceph-{osd-number}
+    ssh {osd-host}
+    sudo mkdir /var/lib/ceph/osd/ceph-{osd-number}
 
-The ``osd_data`` path ideally leads to a mount point with a device that is
-separate from the device that contains the operating system and
-daemons. If an OSD is to use a device other than the OS device, prepare it for
-use with Ceph, and mount it to the directory you just created
+The ``osd_data`` path ought to lead to a mount point that has mounted on it a
+device that is distinct from the device that contains the operating system and
+the daemons. To use a device distinct from the device that contains the
+operating system and the daemons, prepare it for use with Ceph and mount it to
+the directory you just created by running the following commands:
 
 .. prompt:: bash $
 
-	ssh {new-osd-host}
-	sudo mkfs -t {fstype} /dev/{disk}
-	sudo mount -o user_xattr /dev/{hdd} /var/lib/ceph/osd/ceph-{osd-number}
+    ssh {new-osd-host}
+    sudo mkfs -t {fstype} /dev/{disk}
+    sudo mount -o user_xattr /dev/{disk} /var/lib/ceph/osd/ceph-{osd-number}
 
-We recommend using the ``xfs`` file system when running
-:command:`mkfs`.  (``btrfs`` and ``ext4`` are not recommended and are no
-longer tested.)
+We recommend using the ``xfs`` file system when running :command:`mkfs`. (The
```

Comment:
@rzarzynski - Is it still true that we recommend the ``xfs`` file system?

Reply:
unfinished business